src/components/MarkdownProvider/components/CodeExample/index.tsx

```diff
@@ -18,6 +18,11 @@ import { ReactComponent as CodeSandboxIcon } from './assets/codesandbox-icon.svg
 import { retrieveCodesandboxParameters } from './utils/retrieveCodesandboxParameters';
 import { copyToClipboard } from './utils/copyToClipboard';
 
+const StyledCodeBlock = styled(CodeBlock)`
+  border-bottom-left-radius: ${props => props.theme.space.base}px;
+  border-bottom-right-radius: ${props => props.theme.space.base}px;
```

Comment:
These should be based upon `theme.borderRadii.md`. Also, when employing a technique like this, subtract 1 pixel in order to get the right fit. Otherwise, you end up with whitespace between the edges, which can be observed here on zoom.

Reply:
Oo thanks for the tip. The style now uses `theme.borderRadii.md` and gets rid of the whitespace.

Reply:
Weird. Yeah, when that commit comes through, you've got the right styling.